tensorflow_privacy/privacy/membership_inference_attack/data_structures.py

```diff
@@ -529,6 +529,39 @@ def __str__(self):
     ])
 
 
+@dataclass
+class SingleRiskScoreResult:
+  """Results from computing privacy risk scores.
+  this part is quite preliminary: it shows how to leverage privacy risk score to perform attacks with thresholding on risk score
+  """
+
+  # Data slice this result was calculated for.
+  slice_spec: SingleSliceSpec
+
+  train_risk_scores: np.ndarray 
+  
+  test_risk_scores: np.ndarray
+  
+  def attack_with_varied_thresholds(self, threshold_list):
```

Comment:
Can you add a comment for this function?

Reply:
Just added the comment!